artifacts/definitions.py

```diff
@@ -40,7 +40,6 @@
     'IM': 'Instant Messaging / Chat applications artifacts.',
     'iOS' : 'Artifacts related to iOS devices connected to the system.',
     'History Files': 'History files artifacts e.g. .bash_history.',
-    'KnowledgeBase': 'Artifacts used in knowledge base generation.',
```

Comment:
this might be a GRR specific left over we cannot remove yet + @destijl for more insight